src/app/public/modules/action-button/action-button-container.component.scss

```diff
@@ -6,7 +6,7 @@
   display: block;
 }
 
-.sky-action-button-container {
+.sky-action-button-flex {
```

Comment:
Why the rename? Is there a danger of breaking someone's unit tests if they're looking for this selector?

Reply:
The `sky-action-button-container` element is still there - I just moved the flexing declarations to another element in the stack.